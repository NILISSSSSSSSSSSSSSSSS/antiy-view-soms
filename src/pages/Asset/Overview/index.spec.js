import React from 'react'
import { shallow } from 'enzyme'
import AssetOverview from './index'

describe('资产概览 <AssetOverview />', () => {
  let wrapper
  beforeEach(()=>{
    wrapper = shallow(<AssetOverview />)
  })
  it('未知资产列表的正常渲染', () => {
    expect(wrapper.find('.asset-overview')).toHaveLength(1)
  })
  it('快照', ()=>{
    const app = wrapper.debug()
    expect(app).toMatchSnapshot()
  })
})
