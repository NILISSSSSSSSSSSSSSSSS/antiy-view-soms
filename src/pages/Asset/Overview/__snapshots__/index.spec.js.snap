// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`资产概览 <AssetOverview /> 快照 1`] = `
"<div className=\\"asset-overview\\">
  <div className=\\"asset-overview-flex\\">
    <Connect(AssetInformation) />
    <Connect(AssetManufacturer) />
  </div>
  <div className=\\"asset-overview-flex\\">
    <Connect(AssetStatus) />
    <Connect(AssetError) />
  </div>
</div>"
`;
